### PR TITLE
HDDS-11025. Update some ZH docs

### PR DESCRIPTION
**Pull request by houhang1005**
## What changes were proposed in this pull request?

Fix the format in OnPrem.zh.md, current format includes a little problem, guess need fix.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-11025

## How was this patch tested?

Only refer to a markdown document. Without any potential risk.

### CODE REVIEW

File: hadoop-hdds/docs/content/interface/O3fs.md [hunk 1]
```diff
@@ -49,6 +49,58 @@ ozone sh bucket create /volume/bucket
...
+"creationTime" : "2024-06-18T07:25:42.916Z",
+"modificationTime" : "2024-06-18T07:25:42.916Z",
+"sourcePathExist" : true,
+"quotaInBytes" : -1,
+"quotaInNamespace" : -1,
+"bucketLayout" : "FILE_SYSTEM_OPTIMIZED",
+"owner" : "root",
+"link" : false
+} ]
+```
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/interface/O3fs.md:103]: I don't think this level of detail belongs to the documentation about Ozone filesystem interface.  CLI commands are documented in `hadoop-hdds/docs/content/interface/Cli.md`.

```suggestion
```

File: hadoop-hdds/docs/content/interface/O3fs.zh.md [hunk 2]
```diff
@@ -45,6 +45,59 @@ ozone sh bucket create /volume/bucket
...
+"modificationTime" : "2024-06-18T07:25:42.916Z",
+"sourcePathExist" : true,
+"quotaInBytes" : -1,
+"quotaInNamespace" : -1,
+"bucketLayout" : "FILE_SYSTEM_OPTIMIZED",
+"owner" : "root",
+"link" : false
+} ]
+```
+
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/interface/O3fs.zh.md:100]: Same here, please omit this change.

```suggestion
```

File: hadoop-hdds/docs/content/start/OnPrem.zh.md [hunk 3]
```diff
@@ -59,9 +59,9 @@ ozone genconf <path>
   
   {{< highlight xml >}}
     <property>
-        <name>ozone.scm.names</name>
-      <value>scm.hadoop.apache.org</value>
-      </property>
+       <name>ozone.scm.names</name>
+       <value>scm.hadoop.apache.org</value>
+    </property>
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/start/OnPrem.zh.md:64]: This part of the change LGTM.

**Review by adoroszlai — CHANGES_REQUESTED** [review]
Thanks @houhang1005 for working on improving the docs.

File: hadoop-hdds/docs/content/interface/O3fs.zh.md (hunk 2 above)
Comment (houhang1005) [reply hadoop-hdds/docs/content/interface/O3fs.zh.md:100]: Okay, I will fix it.

**Comment by adoroszlai** [comment]
Thanks @houhang1005 for updating the PR.

**Comment by houhang1005** [comment]
Should I rebase all commits to a one commit？

**Comment by adoroszlai** [comment]
> Should I rebase all commits to a one commit？

No, they will be squashed when merging the PR.

**Comment by ivandika3** [comment]
Sorry, my Chinese is not good so I can't review the doc change. Otherwise LGTM.

**Comment by houhang1005** [comment]
> Sorry, my Chinese is not good so I can't review the doc change. Otherwise LGTM.

I see, I will only update the format this time.

**Comment by ivandika3** [comment]
@houhang1005 No, I meant to let other reviewers to take a look. No need to omit the docs change.

**Comment by houhang1005** [comment]
> @houhang1005 No, I meant to let other reviewers to take a look. No need to omit the docs change.

Got it.

**Review by ChenSammi — APPROVED** [review]
Thanks @houhang1005.

**Pull request merged**